### ISSUE TITLE
LogReceiverService: Not able to use Credentials

### ISSUE THREAD

**Issue by Xharze** [NLog/NLog#22]
>When i need to use Credentials ( like in wsHttpBinding - Connection ) there is no way to set them ( only endpointAdress and enpointConfigurationName are available )..

>Workaround is sth. like this:
 
 ```c#
var client = new WcfLogReceiverClient("WSHttpBinding_ILogReceiverServer");
client.ClientCredentials.UserName.UserName = "demo";
client.ClientCredentials.UserName.Password = "demo";
client.ProcessLogMessagesAsync(new NLogEvents());
```

Moved from http://nlog.codeplex.com/workitem/6529

**Comment by 304NotModified** [comment]
I think this was possible in 3.2 already (but broken in 4. 0)

**Comment by kevindaub** [comment]
Is this asking for another constructor that allows for the username and password?

**Comment by 304NotModified** [comment]
It's unclear to me

**Comment by kevindaub** [comment]
This is probably resolved with 4.1. In 4.0, client base was removed, but we added back all the properties like client credentials.

**Comment by 304NotModified** [comment]
I think this is also solved in 4.1
